pages/main-delete.js

```diff
@@ -15,15 +15,18 @@ function view (state, prev, send) {
 
   const header = Header({
     create: () => send('repos:create'),
-    download: (link) => send('repos:download', link)
+    download: (link) => send('repos:clone', link)
   })
 
   return html`
     <body>
       ${sprite()}
       ${header}
       ${Table(archives, send)}
-      ${confirmModal(() => send('repos:deleteConfirm', link))}
+      ${confirmModal(() => send('repos:remove', {
+        confirmed: true,
+        key: link
```

Comment:
having `key` _and_ `link` makes this confusing, do you think we can get it to just use one term across the codebase?